configs/encapsulate_in_http1_connect.yaml

```diff
@@ -4,7 +4,10 @@
 # and running `curl --x 127.0.0.1:10000 https://www.google.com`
 
 admin:
-  access_log_path: /tmp/admin_access.log
+  access_log:
+  - name: envoy.access_loggers.stdoutput
+    typed_config:
+      "@type": type.googleapis.com/envoy.extensions.access_loggers.stdoutput.v3.StdoutputAccessLog
```

Comment:
This change is introducing another subtle change to the config validation which breaks windows and makes this change required.

Now `FileAccessLog` is constructed in the `configuration_impl`. Before it was happening in `source/server/admin/admin.cc`. What this means is that now config validation is also checking that the file points to a valid file path. I think this is an improvement over the status quo although it might be considered a breaking behavior.

Since this is the case now Windows tests are broken because `\temp\envoy` or `\dev\null` does not exist.

Reply:
I think the best way forward is to also add a `NullAccessLogger` and then use that in the configs.

Reply:
> Is this what we want? I would translate to a file access log equivalent 

as i understand - you will be able to specify a fiileaccesslogger and use the unix paths to stdout/stderr/null etc - but this way will be compatible with  non-unix systems too

Reply:
OK I'm fine with whatever you both decide. I agree this is an improvement in general.

Reply:
one comment i would make, altho i suspect its a bit late - im wondering if `v3.StdoutAccessLog` would have been a better name - ive never heard it referred to as stdoutput before

Reply:
Now that you made it not required, how about just removing access logging entirely from all the examples? It will make them cleaner?

Reply:
not sure about the configs *not* in the `/examples` dir but i think for the examples this is broadly a good idea - might need to check that its not required for any specific ones

related - ive opened a ticket to add/improve docs around this #15652

Reply:
let me know if I can help with anything with the issue above.